channeldb/channel.go

```diff
@@ -45,6 +46,13 @@ var (
 	// TODO(roasbeef): flesh out comment
 	openChannelBucket = []byte("open-chan-bucket")
 
+	// outpointBucket stores all outpoints that we were party to and a tlv
```

Comment:
what does "we were party to" mean?

Reply:
Like if we had a channel with this outpoint

Reply:
Changed the comment, lmk what you think